ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4821,6 +4821,60 @@ kong_apis:
      - name: request-size-limiting
        config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: "submitDataExhaustRequest"
+    uris: "{{ data_service_prefix }}/v3/job/request/submit"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/submit"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
+     - name: rate-limiting
+       config.policy: local
+       config.hour: "{{ medium_rate_limit_per_hour }}"
+       config.limit_by: credential
+     - name: request-size-limiting
+       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: "getDataExhaustRequest"
+    uris: "{{ data_service_prefix }}/v3/job/request/read"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/read"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
+     - name: rate-limiting
+       config.policy: local
+       config.hour: "{{ medium_rate_limit_per_hour }}"
+       config.limit_by: credential
+     - name: request-size-limiting
+       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: "listDataExhaustRequest"
+    uris: "{{ data_service_prefix }}/v3/job/request/list"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/list"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
```

Comment:
@sowmya-dixit  it should have dataAdmin ACL

Reply:
Keeping dataAccess as ACL as user can retrieve only his jobs

Reply:
yes, then Access ACL makes sense